lapce-app/src/window_tab.rs

```diff
@@ -1394,6 +1394,39 @@ impl WindowTabData {
                     editor_data.call_hierarchy(self.clone());
                 }
             }
+            RunInTerminal => {
+                if let Some(editor_data) =
+                    self.main_split.active_editor.get_untracked()
+                {
+                    let name = editor_data.get_select_text().content;
```

Comment:
You can use this https://github.com/lapce/lapce/blob/master/lapce-app/src/editor.rs#L2376
to get the selection text which is more reliable.

Reply:
This method retrieves the word under the mouse even if no text is selected. But my intention is to require text selection.

Reply:
There's no harm to retrieves the word under the mouse even if no text is selected.